client/signup/steps/design-type-with-atomic-store/index.jsx

```diff
@@ -187,9 +176,13 @@ class DesignTypeWithAtomicStoreStep extends Component {
 					/>
 				</div>
 				<div className={ designTypeListClassName }>
-					{ this.getChoices().map( this.renderChoice ) }
+					<TileGrid className="design-type-with-atomic-store__list">
```

Comment:
Look like this `className` can be removed.

Reply:
👍  2eedfce893f6ca32fc4db13fb0f48121159b32c7